src/supermarket/config/environments/production.rb

```diff
@@ -116,6 +116,8 @@
   # The value of ENV["DISABLE_HOST_HEADER_ATTACK"] will be parsed as string.
   # Hence we need to convert string to boolean.
   if ActiveModel::Type::Boolean.new.cast(ENV["DISABLE_HOST_HEADER_ATTACK"]) && ENV["ALLOWED_HOST"].present?
-    config.hosts << ENV["ALLOWED_HOST"]
+    ENV["ALLOWED_HOST"].split(",").each do |host|
+      config.hosts << host
```

Comment:
Better to use `host.strip` as someone might keep a whitespace between the domain names after the comma.